packages/synapse-react-client/src/components/AccountLevelBadge/AccountLevelBadge.tsx

```diff
@@ -40,28 +40,35 @@ export const AccountLevelBadge: React.FunctionComponent<
 > = ({ badgeType }: AccountLevelBadgeProps) => {
   const badgeConfig = accountLevelBadgeConfig[badgeType]
   return (
-    <div className={'AccountLevelBadge cardContainer'}>
+    <Card
```

Comment:
Use MUI `Card` to more closely align with design. Move styles from `_account-level-badges.scss` into the component.

Reply:
<img width="791" alt="SRC_updated_card" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/2d10dde2-8d46-41fe-800c-13ee705ace9f">

Reply:
Awesome! Thanks for converting the whole component